MethodsHomework/MethodsHomework.cs

```diff
@@ -166,8 +169,136 @@ public class YourAnswers
     {
         //READ ME
         // You write all your methods here.
+        public static int DoNothing()
+        {
+            return 0 ;
+        }
+
+        public static int ReturnIntWithValue25()
+        {
+            return 25;
+        }
+
+        public static int AddTwoNumbers(int x, int y)
+        {
+            int addIt = x + y;
+            return addIt;
+        }
+
+        public static int IncrementMe(int x)
+        {
+            int y = x++;
+            return y;
+        }
+
+        public static int DecrementMe(int x)
+        {
+            int y = x--;
+            return y;
+        }
+
+        public static int CastDoubleToInt(double x)
+        {
+            double y = x;
+            int d = (int) y;
+            return d;
+        }
+
+        public static double ConvertIntToDouble(int x)
+        {
+            int y = x;
+            double d = (double) y;
+            return d;
+        }
+
+        public static string CastMeIntoAString(double x)
+        {
+            double y = x;
+            string str = x.ToString(); // intellisense is creeping me out
+            return str;
+        }
+
+        public static double ParseMeIntoADouble(string valueToBeParsed)
+        {
+            double result = double.Parse(valueToBeParsed);  // had no been parsed
+            return result; 
+        }
+
+        public static int ReturnMaxIntValue()
+        {
+                return int.MaxValue;
+        }
+
+        public static bool CheckIfIAmTrue(bool amITrue)
+        {
+            if (amITrue != true)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        public static bool ReturnTrueOnlyIfIamBothPositiveAndEven(int x)
+        {
+            if (x >= 0 && x  % 2 == 0)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
         
-    }
+
+        public static int SumTheContentsOfTheArray(int[] array)
+        {
+            int sum = 0;
+            for(int i = 0; i < array.Length; i++)
+                sum += i;
```

Comment:
`i` here is the index (the position in the array). How would you get the value at that position?

(There's more than one way. You can use `i` to get the value, or you can use a different kind of loop to get the value directly.)